### PR TITLE
raw-cpuid: Multiple soundness issues

### PR DESCRIPTION
**Pull request by niklasf**
Draft. To be updated soon with solutions.

### CODE REVIEW

**Comment by Shnatsel** [comment]
Thanks! Looks good overall, we'll be happy to merge once the fixed versions are released. If the maintainers turn out to be unresponsive, we could merge even without having fixed versions.

**Comment by niklasf** [comment]
Point 1 and 3 have now been fixed in v9.0.0. Would it make sense to move point 2 to a separate advisory?

**Comment by Shnatsel** [comment]
Yes, that makes sense.

File: crates/raw-cpuid/RUSTSEC-0000-0000.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+```toml
+[advisory]
+id = "RUSTSEC-0000-0000"
+package = "raw-cpuid"
+date = "2021-01-20"
+url = "https://github.com/RustSec/advisory-db/pull/614"
+categories = ["crash", "memory-corruption"]
```
Comment (Shnatsel) [inline crates/raw-cpuid/RUSTSEC-0000-0000.md:7]: I suggest removing "crash." "category" has to be one of:

"code-execution", "crypto-failure", "denial-of-service", "file-disclosure", "format-injection", "memory-corruption", "memory-exposure", "privilege-escalation"

**Comment by Shnatsel** [comment]
Thank you!

**Pull request merged**